app.rb

```diff
@@ -18,11 +18,12 @@
   `/maestro pause` -- Pauses (or resumes) Spotify playback.
   `/maestro stop` -- Stops playback.
   `/maestro quit` -- Stops playback and quits Spotify.
+  `/maestro restart` -- Quits and restarts Spotify.
 
   `/maestro vol up` -- Increases the volume by 10%.
   `/maestro vol down` -- Decreases the volume by 10%.
   `/maestro vol <amount>` -- Sets the volume to an amount between 0 and 100.
-  `/maestro vol [show]` -- Shows the current Spotify volume.
+  `/maestro vol` -- Shows the current Spotify volume.
```

Comment:
dang I'd just been using `/maestro vol` this whole time anyway - by having been `[show]` (in brackets) would that make it an optional bit?

Reply:
I believe that's how shpotify works, but we don't use it in Maestro, so I removed it from the help text.